partner_compassion/views/partner_compassion_view.xml

```diff
@@ -413,6 +390,5 @@
     <!-- Delete Contacts Top Menu -->
     <delete model="ir.ui.menu" id="mail.menu_contacts"/>
 
-    <menuitem id="sponsorship_compassion.open_customers" action="action_partner_supporter_form"
-              parent="child_compassion.menu_sponsorship_section" sequence="0"/>
+    <menuitem id="sponsorship_compassion.open_customers" action="action_partner_supporter_form" parent="child_compassion.menu_sponsorship_section" sequence="0"/>
```

Comment:
This change is producing your bug in Travis. It is probably due to a wrong merge during rebase.